src/test/java/org/folio/edge/connexion/MainVerticleTest.java

```diff
@@ -35,17 +40,41 @@ public void instanceMainVerticvle() {
     Assert.assertEquals(100, v.getMaxRecordSize());
   }
 
-  @Test
-  public void testAdminHealth(TestContext context) {
-    final int port = 9230;
+  Future<Void> deploy(MainVerticle mainVerticle) {
     JsonObject config = new JsonObject();
-    config.put(SYS_PORT, port);
+    config.put(SYS_PORT, PORT);
+    return vertx.deployVerticle(mainVerticle, new DeploymentOptions().setConfig(config))
+        .mapEmpty();
+  }
 
+  @Test
+  public void testAdminHealth(TestContext context) {
     WebClient webClient = WebClient.create(vertx);
-    webClient.get(port, "localhost", "/admin/health").send().onComplete(context.asyncAssertFailure(f ->
-        vertx.deployVerticle(new MainVerticle(), new DeploymentOptions().setConfig(config))
-            .compose(x -> webClient.get(port, "localhost", "/admin/health").send())
+    deploy(new MainVerticle())
+        .compose(x -> webClient.get(PORT, "localhost", "/admin/health").send())
+        .onComplete(context.asyncAssertSuccess());
+  }
+
+  @Test
+  public void testImportWithUserOK(TestContext context) {
+    NetClient netClient = vertx.createNetClient();
+    deploy(new MainVerticle())
+        .compose(x -> netClient.connect(PORT, "localhost")
+            .compose(socket -> socket.write("U6MyUser")))
+        .onComplete(context.asyncAssertSuccess());
+  }
+
+  @Test
+  public void testImportTooLargeMessage(TestContext context) {
+    MainVerticle mainVerticle = new MainVerticle();
+    mainVerticle.setMaxRecordSize(10);
+    NetClient netClient = vertx.createNetClient();
+    netClient.connect(PORT, "localhost").onComplete(context.asyncAssertFailure(f ->
+        deploy(mainVerticle)
+            .compose(x -> netClient.connect(PORT, "localhost")
+                .compose(socket -> socket.write("00006123456")))
             .onComplete(context.asyncAssertSuccess())
     ));
```

Comment:
```suggestion
    deploy(mainVerticle)
        .compose(x -> vertx.createNetClient().connect(PORT, "localhost"))
        .compose(socket -> socket.write("00006123456"))
        .onComplete(context.asyncAssertSuccess());
```
The same cleanup as in testImportWithUserOK.

Reply:
ok